plenum/test/monitoring/test_revival_spike_resistant_ema_throughput_measurement.py

```diff
@@ -180,7 +210,8 @@ def tm_in_normal(tm_after_start):
         tm.add_request(ts)
 
     # [15, 30) - [225, 240) -- up to 16 not empty windows
-    tm.get_throughput(15)
+    tm.add_request(15)
+    assert tm.get_throughput(15) is None
```

Comment:
This assert should not be a responsibility of this fixture.

Reply:
Done.